test/OpenTelemetry.Tests/Metrics/MetricApiTestsBase.cs

```diff
@@ -178,6 +178,83 @@ public void MetricDescriptionIsExportedCorrectly(string description)
         Assert.Equal(description ?? string.Empty, metric.Description);
     }
 
+    [Fact]
+    public void MetricInstrumentationScopeIsExportedCorrectly()
+    {
+        var exportedItems = new List<Metric>();
+        var meterName = Utils.GetCurrentMethodName();
+        var meterVersion = "1.0";
+        var meterTags = new List<KeyValuePair<string, object>>
+        {
+            new(
+                "MeterTagKey",
+                "MeterTagValue"),
+        };
+        using var meter = new Meter($"{meterName}", meterVersion, meterTags);
+        using var container = this.BuildMeterProvider(out var meterProvider, builder => builder
+            .AddMeter(meter.Name)
+            .AddInMemoryExporter(exportedItems));
+
+        var counter = meter.CreateCounter<long>("name1");
+        counter.Add(10);
+        meterProvider.ForceFlush(MaxTimeToAllowForFlush);
+        Assert.Single(exportedItems);
+        var metric = exportedItems[0];
+        Assert.Equal(meterName, metric.MeterName);
+        Assert.Equal(meterVersion, metric.MeterVersion);
+
+        bool containsMeterTags = metric.MeterTags.Any(kvp =>
+            kvp.Key == meterTags[0].Key && Equals(kvp.Value, meterTags[0].Value));
+        Assert.True(containsMeterTags);
+    }
+
+    [Fact]
+    public void MetricInstrumentationScopeAttributesAreNotTreatedAsIdentifyingProperty()
+    {
+        // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#get-a-meter
+        // Meters are identified by name, version, and schema_url fields
+        // and not with tags.
+        var exportedItems = new List<Metric>();
+        var meterName = "MyMeter";
+        var meterVersion = "1.0";
+        var meterTags1 = new List<KeyValuePair<string, object>>
+        {
+            new(
+                "Key1",
+                "Value1"),
+        };
+        var meterTags2 = new List<KeyValuePair<string, object>>
+        {
+            new(
+                "Key2",
+                "Value2"),
+        };
+        using var meter1 = new Meter(meterName, meterVersion, meterTags1);
+        using var meter2 = new Meter(meterName, meterVersion, meterTags2);
+        using var container = this.BuildMeterProvider(out var meterProvider, builder => builder
+            .AddMeter(meterName)
+            .AddInMemoryExporter(exportedItems));
+
+        var counter1 = meter1.CreateCounter<long>("my-counter");
+        counter1.Add(10);
+        var counter2 = meter2.CreateCounter<long>("my-counter");
+        counter2.Add(10);
+        meterProvider.ForceFlush(MaxTimeToAllowForFlush);
+
+        // The instruments differ only in the Meter.Tags, which is not an identifying property.
+        // The first instrument's Meter.Tags is exported.
```

Comment:
Add an explicit Assert statement to verify that the second Meter's tags are not exported.